tiledb/sm/serialization/meet-capnproto-win32-include-expectations.h

```diff
@@ -0,0 +1,30 @@
+
+// vs2017, capnp_utils.h several layers down includes kj/windows-sanity.h, which
+// *assumes* 'ERROR' will have been #define'd somewhere previously, which in this
+//compilation stream, (with capnp v0.8.0), is *not* true!!! so compilation
+//fails... (appears to also assume  similarly for 'VOID'...)  See if can safely
+// #include <windows.h> here so that the appropriate 'ERROR'/'VOID' entities  will
+// be defined with the correct values...
+#ifdef _WIN32

```

Comment:
Add a header guard similar to our other header files before this statement

Reply:
done